Realm/RLMObjectBase_Private.h

```diff
@@ -26,6 +26,9 @@ NS_ASSUME_NONNULL_BEGIN
 @interface RLMObjectBase ()
 + (void)initializeLinkedObjectSchemas;
 + (bool)isEmbedded;
+// Key Path Strings
+- (void)prepareForRecording;
+@property (nonatomic, nullable) NSMutableArray* lastAccessedNames;
```

Comment:
Asterisks in obj-c code are right-aligned (there's a bunch of these and I won't note them all).